Assets/FieldHandler.cs

```diff
@@ -1,48 +1,70 @@
 ﻿using Fumbbl;
 using Fumbbl.Ffb.Dto;
-using Fumbbl.Ffb.Dto.Reports;
 using Fumbbl.Lib;
 using Fumbbl.Model;
 using Fumbbl.Model.Types;
 using Fumbbl.View;
-using System;
-using System.Collections.Generic;
 using System.Linq;
 using UnityEngine;
-using UnityEngine.UI;
 
 public class FieldHandler : MonoBehaviour
 {
+    private GameObject Ball;
+    private Player HoverPlayer;
+    private RectTransform FieldRect;
+    private ViewObjectList<Player> Players;
+    private ViewObjectList<PushbackSquare> PushbackSquares;
+    private ViewObjectList<TrackNumber> TrackNumbers;
+
     public Camera MainCamera;
-    public GameObject PlayerIconPrefab;
     public GameObject AbstractIconPrefab;
-    public GameObject Field;
-    public GameObject DugoutHome;
-    public GameObject DugoutAway;
-    public GameObject BallPrefab;
     public GameObject ArrowPrefab;
-    public GameObject TrackNumberPrefab;
+    public GameObject BallPrefab;
+    public GameObject DugoutAway;
+    public GameObject DugoutHome;
+    public GameObject Field;
+    public GameObject PlayerCardAway;
+    public GameObject PlayerCardHome;
+    public GameObject PlayerIconPrefab;
     public GameObject ScrollTextPrefab;
+    public GameObject SquareOverlay;
+    public GameObject TrackNumberPrefab;
+    public TMPro.TextMeshProUGUI AwayTeamText;
+    public TMPro.TextMeshProUGUI HomeTeamText;
 
-    public GameObject PlayerCardHome;
-    public GameObject PlayerCardAway;
 
-    public GameObject SquareOverlay;
 
-    public TMPro.TextMeshProUGUI HomeTeamText;
-    public TMPro.TextMeshProUGUI AwayTeamText;
+    ///////////////////////////////////////////////////////////////////////////
+    //  MONOBEHAVIOUR METHODS  ////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////
 
-    private GameObject Ball;
-    private ViewObjectList<Player> Players;
-    private ViewObjectList<PushbackSquare> PushbackSquares;
-    private ViewObjectList<TrackNumber> TrackNumbers;
+    private void OnDestroy()
```

Comment:
I want life cycle methods grouped together. Start() followed by OnDestroy() to make it easier to follow when you read the code.